VirtoCommerce.CatalogModule.Web/Scripts/blades/images-add.js

```diff
@@ -62,6 +64,13 @@ angular.module('virtoCommerce.catalogModule')
                 blade.currentEntities = [];
             };
 
+            $scope.addImageFromUrlHandler = function () {
+                if (blade.useExternalUrl) {
+                    $scope.addImageDirectlyFromUrl()
+                }
+                $scope.addImageFromUrl()
```

Comment:
Should `$scope.addImageFromUrl()` method be executed after 
```
                if (blade.useExternalUrl) {
                    $scope.addImageDirectlyFromUrl()
                }
```
Don't you need an else here?

Reply:
Yes,looks like i missed it.I made a new commit.